src/components/Console.jsx

```diff
@@ -49,13 +49,19 @@ export default function Console({
         className="label console__label"
         onClick={partial(onToggleVisible, currentProjectKey)}
       >
-        Console
-        <span className="console__chevron u__icon">{chevron}</span>
+        <div>
+          Console
+          <span className="console__chevron u__icon">{chevron}</span>
+        </div>
         <span
           className="console__button console__button_clear u__icon"
-          onClick={onClearConsoleEntries}
+          // eslint-disable-next-line react/jsx-no-bind
```

Comment:
Let’s just remove this from `.eslintrc` altogether—`eslint-disable` is good in the case where a generally good rule should be turned off for a special case or false positive. But I basically just don’t think we should have this rule because arrow functions in components aren’t actually bad.